File: app/src/main/java/com/sivakar/eyerefresh/AppDatabase.java
package com.sivakar.eyerefresh;

import androidx.room.Database;
import androidx.room.RoomDatabase;

import com.sivakar.eyerefresh.models.StateLog;

@Database(entities = { StateLog.class}, version = 1, exportSchema = false )
public abstract class AppDatabase extends RoomDatabase {
    public abstract StateLogDao stateLogDao();
}

File: app/src/main/java/com/sivakar/eyerefresh/AppDatabase.kt
package com.sivakar.eyerefresh

import androidx.room.Database
import androidx.room.RoomDatabase
import androidx.room.TypeConverters

@Database(entities = [EventEntity::class], version = 2, exportSchema = false)
@TypeConverters(Converters::class)
abstract class AppDatabase : RoomDatabase() {
    abstract fun eventDao(): EventDao
} 